packages/angular_devkit/build_angular/src/builders/browser/specs/styles_spec.ts

```diff
@@ -655,4 +655,26 @@ describe('Browser Builder styles', () => {
     result = await browserBuild(architect, host, target, { optimization: true });
     expect(await result.files['styles.css']).toContain('rgba(0,0,0,.15)');
   });
+
+  it('works when using the same css file in `styles` and `stylesUrl`', async () => {
+    host.writeMultipleFiles({
+      'src/styles.css': `
```

Comment:
This is used as a global style by default right? Is there some way we can explicitly list this as a global file as part of this test? As is, we're implicitly depending on this default which could change and invalidate the intent of the test but probably leave it still passing.

Reply:
Yes, that is correct.